src/main/ngapp/accio/src/app/editor/magic-wand.service.ts

```diff
@@ -298,12 +212,54 @@ export class MagicWandService {
         }
         // Visits the pixel and check if it should be part of the mask.
         visited.add(this.coordToDataArrayIndex(x, y, imgData.width));
-        if (this.getIsScribbleMask(scribbles, imgData, neighborPixel, tolerance)) {
+        // if (this.getIsScribbleMask(scribbles, imgData, neighborPixel, tolerance)) {
+        //   visit.push(neighborPixel);
+        // }
```

Comment:
If this code is unused, you should just remove it. If you need to refer to it later, you can always look in the git history.